pkg/cmdutil/userio/interop.go

```diff
@@ -12,15 +13,16 @@ type nonInteractiveHandler struct {
 }
 
 func InfoLog(message string) string {
-	style := newNonInteractiveStyles()
-	return fmt.Sprintf("%s %s", style.infoStyle.Render("INFO:"), message)
+	style := NewNonInteractiveStyles()
```

Comment:
I've added `styles` field to the struct, so you can reuse it instead of reinitializing it. Same for `WarnLog`